src/propTypeShapes.js

```diff
@@ -27,3 +27,8 @@ export const RouterPropTypeShapes = {
     goForward: PropTypes.func,
   }),
 }
+export const tooltipPropType = PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.oneOf([false])])
+
+export const requiredTooltipPropType = PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired
```

Comment:
I agree that is overkill if the difference is only the isRequired, but in this case there is one more difference and
```
export const tooltipPropType = PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.oneOf([false])])
```
the difference is for not required tooltip false should be allowed if we want to use AND/OR statements : 
`prop={condition1 && value}`
otherwise it will throw an error like in this [case](https://github.com/oVirt/ovirt-web-ui/pull/1233#discussion_r496452236)

Reply:
@rszwajko ?

Reply:
Ok. Let's go this path and treat `false` | `undefined` | `'default'` the same.

Reply:
IMO creating these special types is unnecessary. We can specify these types directly in appropriate files, without importing anything. I would better use this approach in other cases, for more complicated proptypes specification.